### ISSUE TITLE
Expand to the seasonal events (Solar Terms)

### ISSUE THREAD

**Issue by Redstone-ray** [casperge/RealisticSeasons-issues#20]
Just bring over my suggestion on discord:

Currently, each season has a very "direct" transition into the next season. For example, the leaves immediately change to yellow after summer ends, and snow immediately starts to pile up when winter arrives, regardless that it is still in early November.  

With the recent addition of the calendar system, I think there could be an add-on event system that brings gradual transition into each season. For example, the snow melts on a specific date, then the sky gradually lightens, and then flowers begin to blossom. In the end, the bees get spawned in. 

Reference the “solar terms”https://en.wikipedia.org/wiki/Solar_term, it includes solar terms such as “Insects awaken”, and can translate into “Bee spawning” or “Light Snow” corresponding to “snow start to appear on the ground”.  I think this feature can bring a more realistic feeling to the players and can allow future customization of the plugin.

**Comment by casperge** [comment]
Good idea, Wikipedia page helps a lot. Would be a good quality improvement and I could certainly add this except for gradual colour change (custom biome limits of servers would be reached)
